test/Libraries/RevitIntegrationTests/SelectionTests.cs

```diff
@@ -377,6 +377,56 @@ public void SelectFaces()
             Assert.AreEqual(0, list.Count);
         }
 
+        [Test]
+        [Category("SmokeTests")]
+        [TestModel(@".\Selection\Selection.rfa")]
+        public void SelectionVerifyElementID()
+        {
```

Comment:
This test:
- opens an xml graph w/ a model element selection node
- verifies the selectionId and node UUID
- saves to temporary json file
- closes and reopens the graph
- verifies data remains unchanged after execution
- deletes temporary json file

@smangarole